### PR TITLE
Added the goodest girl.jpeg

### PR DESCRIPTION
**Pull request by Scientist8**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -424,6 +424,7 @@ <h1>MAXIMIZE YOUR WIN</h1>
           <a href="/embr" style="color: #b8bb26">Embr</a>
         </p>
       </div>
+      <img src="static/images/bobi.jpg" id="bobi" style="width: 480px; height: 640px;">
```
Comment (eigenlenk) [inline index.html:427]: could we introduce a gallery component here?

Comment (Scientist8) [reply index.html:427]: I have actually no idea what I'm doing and don't know a gallery component is :( you can edit it however you want mate

**Comment by ricberht** [comment]
Can you add another picture of Bobi please? I think we only have half as many photos of Bobi as we need.

File: index.html (hunk 1 above)
Comment (ricberht) [reply index.html:427]: a gallery component is a mechanism to browse a wide array of images

Comment (Scientist8) [reply index.html]: Thanks for the reply richbert! After a bit of searching on the internet and talking with gpt i made a small gallery, hope it works.

**Comment by Scientist8** [comment]
> Can you add another picture of Bobi please? I think we only have half as many photos of Bobi as we need.

Haha I definitely will once get the hang of things in general :' D

**Review by Dishpit — APPROVED** [review]
nice

**Pull request merged**